docs/application/native/guides/device/diagnostics.md

```diff
@@ -156,10 +210,52 @@ void request_data()
 
         fwrite(buff, 1, bytes_read, stdout); // write received data to the STDOUT
     }
+
+    diagnostics_destroy(ctx);
 }
 ```
 
-If `diagnostics_request_client_data()` returns successfully, the obtained data can be read by `diagnostics_data_read()`.
+## Stability-monitor
+
+Stability monitor provides interface `org.tizen.stability-monitor` for getting process specific data such as:
```

Comment:
Here it is Stability monitor,
in line 218 it is with a hyphen ## Stability-monitor

Can we avoid the hyphen, I understand there is a hyphen here `org.tizen.stability-monitor` but consistency sake either use with hyphen in heading and sentence or without.